### DISCUSSION TITLE
Terrace irrigation

### DISCUSSION THREAD

**Discussion by diyanei** [jeroenterheerdt/HAsmartirrigation#243] [General]
Hi @jeroenterheerdt  and HA smartirrigation users/devs !

First off, thank you so much for this integration I now use for 2 years+ and which definitively did help my irrigation routine so much, I almost only kept the (mandatory) filling of my tank.

You could find a quick description  of my installation [here](https://diyanei.github.io/ha/irrigation#big-picture).

Anyhow, I was just wondering if you had general advice on how to configure HAsmartirrigation to take into account the facts that:
- all the plants are potted, on a terrace (no roof)
- the watering system is a drip 
- one zone, one source of watering, one pump

Currently, as I obviously ended up with a working configuration, I actually estimated the surface to water and made some "test driven configuration" until I considered it correct ...
I am wondering if anyone with better knowledge on this kind of situation had good advice configuration wize.

**Comment by jeroenterheerdt** [comment]
first of all, thanks for checking in and helping out with the French translations! Even for a drip system you should be able to find the throughput per minute. Once you have that in liters per minute you can estimate how much soil there is by measuring your pots. Then, this is where it gets tricky, because you're not watering grass and the PyETO module kind of expects you're watering grass. I am not sure the module it uses allows you to configure anything else, but that would be nice feature request (hint hint - please create one).
So I'd expect you'd need to lower the amount of soil or increase the throughput to bring down the duration to avoid flooding your plants.
I'd also leverage the maximum bucket size feature of 2.0 and have one "real" zone and multiple "fake" zones with different combinations of sensor groups to figure out what works best for you.

Hope this helps a bit!
